core-feature-pack/common/src/main/resources/content/bin/systemd/generate_systemd_unit.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/sh
+
+# This script is just a helper to generate the systemd unit files for WildFly
+# assuming that the server home will be the one that hold this script file.
+# It also allows to specify the user and group that will run the server.
+
+function print_usage() {
+  echo "INFO: Usage: "
+  echo "${SCRIPT_NAME} -m,--mode standalone|domain [-u,--user user] [-g,--group group] [-c,--confirm y|n]"
+
+  exit 1
+}
+
+SCRIPT_NAME=$(basename "${0}")
+SCRIPT_DIR="$(dirname "$(realpath "${0}")")"
+JBOSS_HOME="$(realpath "${SCRIPT_DIR}/../..")"
+JBOSS_SYSTEMD_MODE="${1}"
+
+JBOSS_SYSTEMD_MODE=""
+JBOSS_SYSTEMD_USER="wildfly"
+JBOSS_SYSTEMD_GROUP="wildfly"
+JBOSS_CONFIRM="n"
+
+while [ "$#" -gt 0 ]
```

Comment:
I did not saw it on the first pass, but all the variables in the args treatement are "$XXX" instead of "${XXX}". For consistency purpose and respecting best practices, it might be better to fix that.

Reply:
Done